spec/features/admin_claim_reject_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.feature "Rejecting a claim" do
+  let(:user_id) { "userid-345" }
+
+  context "when a user is logged in as a service operator" do
+    before do
+      stub_dfe_sign_in_with_role(AdminSession::SERVICE_OPERATOR_DFE_SIGN_IN_ROLE_CODE, user_id)
+      visit admin_path
+      click_on "Sign in"
+    end
+
+    scenario "they can reject a claim" do
+      freeze_time do
```

Comment:
Is this `free_time` actually necessary? I can't see anything in the spec that is checking timestamps.